src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -304,6 +304,39 @@ public void execute(Boolean succeeded)
       return deps.toArray(new Dependency[deps.size()]);
    }
    
+   public void withDataImportSAV(String userAction, final Command command)
+   {
+     withDependencies(
+        "Preparing Import from SAV",
```

Comment:
Yes, makes sense. SPSS is the product name, SAV the file format. Originally I thought of listing only extension names, but products is more clear in this case.